CHANGELOG.md

```diff
@@ -45,6 +45,10 @@ Version 4.0.0 is a backwards-incompatible release. Please see the [upgrade instr
 - Add on_gke submodule [#182]
 - Add `excluded_resources` variable to forseti_conf_server.yaml. [#213]
 
+### Added
```

Comment:
(minor) Can you please move this change to the top of the changelog (line 9) and add it to the unreleased version. Looks good though, thanks for adding this!

`
## [Unreleased]

### Added

- Add Stackdriver monitoring agent to Forseti VMs [#177]

`

Reply:
@ocervell I apologize for the long delay.  Have you been able to review @gkowalski-google comments?  I'd be happy to get this merged into the next module release (5.1).